packages/mirror/src/Mirror.ts

```diff
@@ -44,6 +48,102 @@ export const getMintIntent = async (
   return transaction
 }
 
+export const getProjectFees = async (
+  mint: MintActionParams,
+): Promise<bigint> => {
+  const { chainId, contractAddress } = mint
+  const client = createPublicClient({
+    chain: chainIdToViemChain(chainId),
+    transport: http(),
+  })
+
+  // get treasuryConfiguration address
+  const treasuryConfiguration = await client.readContract({
+    address: contractAddress,
+    abi: [
+      {
+        inputs: [],
+        name: 'treasuryConfiguration',
+        outputs: [
+          {
+            internalType: 'address',
+            name: '',
+            type: 'address',
+          },
+        ],
+        stateMutability: 'view',
+        type: 'function',
+      },
+    ],
+    functionName: 'treasuryConfiguration',
+  })
+
+  // get feeConfiguration address
+  const feeConfiguration = await client.readContract({
+    address: treasuryConfiguration,
+    abi: [
```

Comment:
Same here - you can even drop both of them in the same ABI like `MIRROR_FEE_ABI` or something.